UnitTestProject1/Problem1Tests.cs

```diff
@@ -0,0 +1,196 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Problems.UnitTests
+{
+    [TestClass]
+    public class Probem1Tests
+    {
+        Problem1 problem;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            problem = new Problem1();
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_EmptyArray_Zero()
+        {
+            int[] arr = { };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_OnlyZero_SumIsZero()
+        {
+            int[] arr = { 0 };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_AllPositiveNoEvenNumbers_SumIsZero()
+        {
+            int[] arr = { 1, 3, 52241, 1201, 11 };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_AllPositiveNoEvenNumbersDuplicates_SumIsZero()
+        {
+            int[] arr = { 1, 3, 3, 3, 1 };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 0);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_AllPositiveNoOddNumbers_SumIs100()
+        {
+            int[] arr = { 4, 28, 0, 60, 8 };
+
+            long resultingSum = problem.SumOfEvenNumbers(arr);
+
+            Assert.AreEqual(resultingSum, 100);
+        }
+
+        [TestMethod]
+        public void SumOfEvenNumbers_AllPositiveNoOddNumbersDuplicates_SumIs40()
```

Comment:
Again this is redundant with the above test.

Reply:
As with the previous case of redundant tests for duplicated numbers, I've made a few changes at 
6703401 to account for the observation.